src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -1,63 +1,62 @@
-import merge from 'deepmerge';
+/* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import dataLink from './demo/data--link';
 import dataButton from './demo/data--button';
 import dataRemovable from './demo/data--removable';
 import tag from './ecl-tag.html.twig';
 import notes from './README.md';
 
+// Preserve the adapted specs.
+const prepareTag = data => {
+  const typeList = {};
+  typeList[data.tag.type] = data.tag.type;
+  data.variant = select(
+    'tag.type',
+    typeList,
+    data.tag.type,
+    buttonLabels.required
+  );
+  data.tag.label = text('tag.label', data.tag.label, buttonLabels.required);
+  if (data.tag.path) {
+    data.tag.path = text('tag.path', data.tag.path, buttonLabels.required);
+  }
+  if (data.tag.aria_label) {
+    data.tag.aria_label = text(
+      'tag.aria_label',
+      data.tag.aria_label,
+      buttonLabels.required
+    );
+  }
+  if (data.default_icon_path) {
+    const iconPathList = {};
```

Comment:
This is a pattern, you did the same  everywhere and you can simplify this, in this case it would be:
```
data.default_icon_path = select(
      'default_icon_path',
      [defaultSprite],
      defaultSprite,
      buttonLabels.required
    );
```